lib/pointer.js

```diff
@@ -189,6 +189,25 @@ Pointer.parse = function (path, originalPath) {
   return pointer.slice(1);
 };
 
+
+/**
+ * Like `Pointer.parse` but only returns the length of the pointer.
```

Comment:
Hmm, this naming is very confusing.
By `length` one could think of an actual length of a string, while what we actually verify here is the level / depth.